src/math2d/regioned_angle.rs

```diff
@@ -0,0 +1,111 @@
+use std::f32::consts::PI;
+
+use super::{line::Line, point::Point, vector::Vector};
+
+pub trait RegionedAngleTrait {
+    fn get_radians(&self) -> f32;
+    fn degrees(&self) -> f32;
+    fn min_degrees(&self) -> i32;
+    fn max_degrees(&self) -> i32;
+}
+
+#[derive(Debug, Clone, Copy, PartialEq, PartialOrd)]
+pub struct RegionedAngle<const MIN_DEGREES: i32, const MAX_DEGREES: i32> {
+    radians: f32,
+}
+
+impl<const MIN_DEGREES: i32, const MAX_DEGREES: i32> RegionedAngle<MIN_DEGREES, MAX_DEGREES> {
```

Comment:
I like your choice to spell it capital case, that way it is easy to distinguish.